packages/server/src/workers/index.ts

```diff
@@ -2,6 +2,7 @@ import { Resource } from '@medplum/fhirtypes';
 import { MedplumRedisConfig } from '../config';
 import { logger } from '../logger';
 import { addDownloadJobs, closeDownloadWorker, initDownloadWorker } from './download';
+import { addScheduledTimingJobs, closeScheduledTimingWorker, initScheduledTimingWorker } from './scheduledtiming';
```

Comment:
Now that we renamed `Bot.scheduledTiming` to `Bot.cron[x]`, can we rename the worker file to `cron.ts`?

`addScheduledTimingJobs` -> `addCronJobs`
`initScheduledTimingWorker` -> `initCronWorker`
`closeScheduledTimingWorker` -> `closeCronWorker`